### PR TITLE
#532 possibility to clear SequenceAllocation cache

### PR DESCRIPTION
**Pull request by mdii**
Changes:
- PolyJDBC upgrated to 0.6.6
- Exposed JaversSqlRepository as an Spring Bean

With the new method in JaversRepository, `JaversSqlRepository.evictCache()` can be misleading. I am wondering if we can rename it to something like `JaversSqlRepository.evictGlobalIdCache()`.

### CODE REVIEW

File: javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java [hunk 1]
```diff
@@ -53,26 +57,29 @@
...
+    @Bean
     @ConditionalOnMissingBean
-    public Javers javers(ConnectionProvider connectionProvider,
-                         PlatformTransactionManager transactionManager) {
-        JaversSqlRepository sqlRepository = SqlRepositoryBuilder
-                .sqlRepository()
-                .withConnectionProvider(connectionProvider)
-                .withDialect(javersSqlDialectName())
-                .build();
+    public JaversSqlRepository javersSqlRepository(ConnectionProvider connectionProvider) {
```
Comment (bartoszwalacik) [inline javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java]: not sure why this change? is it only a reformatting?

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/JaversSqlRepository.java [hunk 2]
```diff
@@ -131,4 +136,11 @@ public int getGlobalIdPkCacheSize(){
     public SqlRepositoryConfiguration getConfiguration() {
         return sqlRepositoryConfiguration;
     }
+
+    /**
+     * @since 3.1.1
```
Comment (bartoszwalacik) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/JaversSqlRepository.java]: maybe some javadoc for javers users? at least a link to the github issue

**Comment by bartoszwalacik** [comment]
looks great! please take a look at my 2 comments

File: javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java (hunk 1 above)
Comment (mdii) [inline javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java]: Because we need to expose JaversSqlRepository as a bean to be able to call its `evictSequenceAllocationCache()`.

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/JaversSqlRepository.java (hunk 2 above)
Comment (mdii) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/JaversSqlRepository.java]: Sure, I will do it.
